leetduel-frontend/src/app/page.tsx

```diff
@@ -12,20 +18,22 @@ function HomeContent() {
   const [username, localSetUsername] = useState("");
   const [message, setMessage] = useState("");
   const [members, setMembers] = useState<string[]>([]);
-  const [joined, setJoined] = useState(false);
+  const [partyStatus, setPartyStatus] = useState<PartyStatus>(PartyStatus.UNJOINED);
   const [localPartyCode, setLocalPartyCode] = useState("");
   const [timeLimit, setTimeLimit] = useState("");
   const [easy, setEasy] = useState(true);
   const [medium, setMedium] = useState(true);
   const [hard, setHard] = useState(true);
+  const [showBanner, setShowBanner] = useState(false);
+  const [goodBanner, setGoodBanner] = useState(true);
```

Comment:
a field for whether to display an error or not. the banner will appear as blue if `goodBanner` is true, and red if otherwise.